google-cloud-language/README.md

```diff
@@ -13,30 +13,18 @@ steps:
 
 1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
 2. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
-3. [Enable the Google Cloud Natural Language API.](https://console.cloud.google.com/apis/library/language.googleapis.com)
+3. [Enable the Cloud Natural Language API.](https://console.cloud.google.com/apis/library/language.googleapis.com)
 4. [Setup Authentication.](https://googleapis.github.io/google-cloud-ruby/#/docs/google-cloud/master/guides/authentication)
 
 ### Installation
 ```
 $ gem install google-cloud-language
 ```
 
-### Preview
-#### LanguageServiceClient
-```rb
-require "google/cloud/language"
```

Comment:
Is the removal of this example from the protos? Or was this something we added manually?

Reply:
@googleapis/yoshi-ruby Anyone know why these code examples were removed? Is that intentional?

Reply:
AFAICT it was originally a manual addition.

Reply:
Should it part of the synth.py file then?

Reply:
Ideally it should be generated. I'm hesitant to insert a hand-coded sample into the generation process as it would seem up-to-date without necessarily actually being so.

Reply:
@dazuma Thoughts on losing the code example from the README and method documentation?

Reply:
It's doubtful it was a manual addition, or synth would have killed it long before now. My understanding is that these samples (along with generated acceptance tests) are generated from a gapic config, and it was removed from there for some reason (https://github.com/googleapis/googleapis/commit/d05a746ba27b89374c7407c88c0d5dabfb0afc7c#diff-5e697703cdd806bbb323023b3b6a47e2L27).

I'm also unhappy about losing the code sample, but sample generation in the docs is something I'd rather figure out in the microgenerators rather than patch things here.

Reply:
K, sounds good. Thanks for the clarification.